src/PartKeepr/ProjectBundle/Entity/Report.php

```diff
@@ -103,8 +103,11 @@ public function getName()
      */
     public function setName($name)
     {
-        $this->name = $name;
-
+        if ($name != NULL) {
+            $this->name = $name;
+        } else {
+            $this->name = 'NewReport';
```

Comment:
Can you add i18n please? Otherwise once i18n will be implemented, we would never find that spot again

Reply:
Just a comment with // @todo i18n is enough